package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "3.0.1",
+  "version": "3.1.0",
```

We bump versions automatically, so I'll take this out.

```suggestion
  "version": "3.0.1",
```